src/libs/Navigation/AppNavigator/createCustomStackNavigator/CustomRouter.ts

```diff
@@ -1,39 +1,40 @@
-import _ from 'underscore';
-import {StackRouter} from '@react-navigation/native';
+import {StackRouter, RouterConfigOptions, ParamListBase, StackNavigationState, PartialState} from '@react-navigation/native';
 import NAVIGATORS from '../../../../NAVIGATORS';
+import {ResponsiveStackNavigatorRouterOptions} from '../types';
 
-/**
- * @param {Object} state - react-navigation state
- * @returns {Boolean}
- */
-const isAtLeastOneCentralPaneNavigatorInState = (state) => _.find(state.routes, (r) => r.name === NAVIGATORS.CENTRAL_PANE_NAVIGATOR);
+type MutableState = {
+    index: number;
+    stale: true;
```

Comment:
```suggestion
    stale: boolean;
```

Reply:
@kubabutkiewicz With a boolean it gives a type error, but I agree there should be a better way, I'll think about it
![image](https://github.com/Expensify/App/assets/23176449/2e6dc111-421d-4c50-89d0-69eeaf4e1d9c)